nullaway/src/test/java/com/uber/nullaway/EnsuresNonNullTests.java

```diff
@@ -48,20 +48,16 @@ public void requiresEnsuresNonNullStaticFields() {
             "class Foo {",
             "  @Nullable static Item nullableItem;",
             "  @RequiresNonNull(\"nullableItem\")",
-            "  // BUG: Diagnostic contains: For @RequiresNonNull annotation, cannot find instance field",
             "  public static void run() {",
-            "    // BUG: Diagnostic contains: dereferenced expression nullableItem is @Nullable",
             "    nullableItem.call();",
             "     ",
             "  }",
             "  @RequiresNonNull(\"this.nullableItem\")",
             "  // BUG: Diagnostic contains: For @RequiresNonNull annotation, cannot find instance field",
             "  public static void test() {",
-            "    // BUG: Diagnostic contains: dereferenced expression nullableItem is @Nullable",
             "    nullableItem.call();",
             "  }",
             "  @EnsuresNonNull(\"nullableItem\")",
-            "  // BUG: Diagnostic contains: For @EnsuresNonNull annotation, cannot find instance field",
```

Comment:
Do we have some other test that checks that we get errors when writing `@EnsuresNonNull` or `@RequiresNonNull` on non-existent fields?

Reply:
Added a couple more